queenbee/schema/workflow.py

```diff
@@ -88,6 +89,19 @@ def visualize(self):
         # call index.html
         os.system(os.path.join(directory, 'chart/index.html'))
 
+    def to_diagraph(self, filename=None):
+        """Return a graphviz instance of a diagraph from workflow"""
+        if filename is None:
+            filename = self.id
+        f = Digraph(self.name, filename='{}.gv'.format(filename))
```

Comment:
Now that we are using this library should we remove the other visualization option?

Reply:
I think so.